Run lexer panics if last line is non-empty

If the last line of my `Runfile` isn't empty, run throws the following error:

```
panic: Lexer.Next: No rune available

goroutine 1 [running]:
github.com/tekwizely/go-parsing/lexer.(*Lexer).Next(0xc0000cc1e0, 0x521c00)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/lexer@v0.0.0-20190714215300-5be83bb42370/lexer.go:159 +0x102
github.com/tekwizely/run/internal/lexer.lexCmdScriptLine(0xc0000bc020, 0xc0000cc1e0, 0x4f31e0)
        /workspace/run/internal/lexer/lexer.go:690 +0xe4
github.com/tekwizely/run/internal/lexer.(*LexContext).lex(0xc0000bc020, 0xc0000cc1e0, 0x4e91c0)
        /workspace/run/internal/lexer/lexer.go:41 +0x7e
github.com/tekwizely/go-parsing/lexer.(*tokenNexter).hasNext(0xc0000bc040, 0x7f6291d24460)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/lexer@v0.0.0-20190714215300-5be83bb42370/tokennexter.go:55 +0x9d
github.com/tekwizely/go-parsing/lexer.(*tokenNexter).Next(0xc0000bc040, 0x0, 0x0, 0xc0000a43d0, 0xc0000aca20)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/lexer@v0.0.0-20190714215300-5be83bb42370/tokennexter.go:22 +0x2f
github.com/tekwizely/go-parsing/parser.(*Parser).growPeek(0xc0000d0050, 0x1, 0x0)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/parser@v0.0.0-20190714215300-5be83bb42370/parser.go:199 +0x75
github.com/tekwizely/go-parsing/parser.(*Parser).CanPeek(...)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/parser@v0.0.0-20190714215300-5be83bb42370/parser.go:59
github.com/tekwizely/run/internal/parser.expectCmdScript(0xc0000bc080, 0xc0000d0050, 0xc0000a6540, 0x5, 0x0)
        /workspace/run/internal/parser/parser.go:630 +0x41c
github.com/tekwizely/run/internal/parser.tryMatchCmd(0xc0000bc080, 0xc0000d0050, 0xc0000d6000, 0xc0000c6000)
        /workspace/run/internal/parser/parser.go:238 +0x149
github.com/tekwizely/run/internal/parser.parseMain(0xc0000bc080, 0xc0000d0050, 0x4f3240)
        /workspace/run/internal/parser/parser.go:200 +0xa45
github.com/tekwizely/run/internal/parser.(*parseContext).parse(0xc0000bc080, 0xc0000d0050, 0x4ff720)
        /workspace/run/internal/parser/parser.go:46 +0x7f
github.com/tekwizely/go-parsing/parser.(*astNexter).hasNext(0xc0000bc0a0, 0xc0000d0050)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/parser@v0.0.0-20190714215300-5be83bb42370/astnexter.go:58 +0x97
github.com/tekwizely/go-parsing/parser.(*astNexter).Next(0xc0000bc0a0, 0xc0000bc040, 0xc0000a4370, 0x53c700, 0xc0000bc0a0)
        /workspace/go/pkg/mod/github.com/tekwizely/go-parsing/parser@v0.0.0-20190714215300-5be83bb42370/astnexter.go:30 +0x2f
github.com/tekwizely/run/internal/parser.Parse(0xc0000bc020, 0x1e)
        /workspace/run/internal/parser/parser.go:79 +0x133
main.main()
        /workspace/run/main.go:125 +0x507
```

--- a/internal/lexer/lexer.go
+++ b/internal/lexer/lexer.go
@@ -686,7 +686,7 @@ func lexCmdScriptLine(_ *LexContext, l *lexer.Lexer) LexFn {
 	// We have a script line
 	// Consume the full line, including eol/eof
 	//
-	for !matchNewline(l) {
+	for !matchNewlineOrEOF(l) {
 		l.Next()
 	}
 	l.EmitToken(TokenScriptLine)

